src/Component/FileImportButton.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+
+type FileImportButtonProps = {
+  id: string;
+  buttonText: string;
+  fileType: string;
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;

```

Comment:
Is this actually optional? What's the use-case/functionality if it weren't passed in?

Reply:
Updated

I had originally left it optional in case nothing was passed, but you're right, there's no use-case for this component if we don't handle the file